src/java.base/share/classes/jdk/internal/access/foreign/MemorySegmentProxy.java

```diff
@@ -27,6 +27,7 @@
 package jdk.internal.access.foreign;
 
 import jdk.internal.misc.ScopedMemoryAccess;
+import jdk.internal.vm.annotation.ForceInline;
```

Comment:
Is this import needed?

Reply:
Leftover form past experiments - good catch!